### ISSUE TITLE
Specify tab length

### ISSUE THREAD

**Issue by mrapogee** [js2coffee/js2coffee#374]
Title says it all, allow for choosing between 2-4 spaces for tab length. Really annoying having to convert it with a vim macro.

**Comment by hanneskaeufler** [comment]
According to the usage instructions this is an option: https://github.com/js2coffee/js2coffee/blob/master/Usage.txt#L7

**Comment by mrapogee** [comment]
I should have specified that I was suggesting this for the web UI. Should I have posted some where else?

**Comment by rstacruz** [comment]
nah, this is the right place too, since the web version is simply the gh-pages branch of this repo.

**Comment by eliasthompson** [comment]
This would be fantastic. Seconded.

**Comment by mrapogee** [comment]
Any update on this? I'll implement it if someone can point me in the right direction on how to get started. I'm sure I could figure it out but would be a huge help from someone familiar with the source.

**Comment by rstacruz** [comment]
That would be in the [gh-pages branch](https://github.com/js2coffee/js2coffee/tree/gh-pages).

**Comment by mrapogee** [comment]
Here's the pull request:

https://github.com/js2coffee/js2coffee/pull/382
